src/components/NewSnippet.jsx

```diff
@@ -30,6 +30,12 @@ class NewSnippet extends React.Component {
         e.preventDefault();
       }
     };
+    this.recalcLangHeaderHeight = () => {
+      const newSnippetHeaderHeight = document.getElementsByClassName('new-snippet-code-header')[0].offsetHeight;
```

Comment:
Despite not having concerns regarding this approach I want to share that I noticed and that I don't like much.

1. I noticed that it's allowed to use "space" (` `) in tags while backend doesn't support them. It was done intentionally,  tags must not contain few words.

2. It's kind weird that once I hit enter to add a tag, my input carriage moved down to the next line. Usually on websites it's on the same line unless there's no more space.

What do you think?

Reply:
I think we can ad additional tag validation on client side. and as for 2 point I can take a look and see what I can do with it